plugins/org.eclipse.emf.mwe2.language.ide/src-gen/org/eclipse/emf/mwe2/language/ide/AbstractMwe2IdeModule.java

```diff
@@ -61,4 +50,9 @@ public void configureContentAssistLexer(Binder binder) {
 		return IRenameStrategy2.DefaultImpl.class;
 	}
 	
+	// contributed by org.eclipse.xtext.xtext.generator.parser.antlr.ex.ExternalAntlrLexerFragment
+	public void configureContentAssistLexer(Binder binder) {
```

Comment:
I'm not sure yet why we do generate FQN here, but that is probably something to be tuned on the fragment.